packages/taco/src/conditions/schemas/return-value-test.ts

```diff
@@ -1,11 +1,22 @@
 import { z } from 'zod';
 
-import { paramOrContextParamSchema } from './context';
+import {
+  paramOrContextParamSchema,
+  rpcParamOrContextParamSchema,
+} from './context';
 
 export const returnValueTestSchema = z.object({
   index: z.number().int().nonnegative().optional(),
   comparator: z.enum(['==', '>', '<', '>=', '<=', '!=']),
   value: paramOrContextParamSchema,
 });
 
+export const rpcReturnValueTestSchema = z.object({
+  index: z.number().int().nonnegative().optional(),
+  comparator: z.enum(['==', '>', '<', '>=', '<=', '!=']),
+  value: rpcParamOrContextParamSchema,
+});
+
```

Comment:
Need to think on this some more.

At the very least we should have both of these schemas extend from a common schema that has `index` and `comparator`; that way only `value`  would be different and we would have `index` and `comparator` repeated